seqr/views/apis/variant_search_api_tests.py

```diff
@@ -308,7 +308,11 @@ def test_query_all_projects_variants(self, mock_get_variants):
         def _get_variants(results_model, **kwargs):
             results_model.save()
             self.assertSetEqual(expected_searched_families, {f.guid for f in results_model.families.all()})
-            return deepcopy(VARIANTS), len(VARIANTS)
+            matched_variants = [
```

Comment:
this test was mocking out an impossible case - a user with no access to a project getting variants back associated with families in that project. this does not reflect a change in functionality, but rather makes the mocked case more accurate